dz10/dz10/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dz10
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //1zad
+            Console.Write("Введите натуральное число n: ");
+            int n = int.Parse(Console.ReadLine());
+
+            Console.Write("Введите действительное число x: ");
+            double x = double.Parse(Console.ReadLine());
+
+            double sum = 0.0;
+            for (int i = 0; i <= n; i++)
+            {
+                double term = (double)(i + 1) / (i + 2) * Math.Pow(x, i);
+                sum += term;
+            }
```

Comment:
В задании первое слагаемое 1, а у вас 1/2 (i = 0).
```suggestion
            double sum = 1.0;
            for (int i = 1; i <= n; i++)
            {
                double term = (double)(i + 1) / (i + 2) * Math.Pow(x, i);
                sum += term;
            }
```